scripts/build/Platform/Android/build_config.json

```diff
@@ -81,15 +81,15 @@
     ],
     "COMMAND":"../Windows/build_asset_windows.cmd",
     "PARAMETERS": {
-        "CONFIGURATION":"profile",
-        "OUTPUT_DIRECTORY":"build\\windows",
-        "CMAKE_OPTIONS":"-DCMAKE_SYSTEM_VERSION=10.0",
-        "CMAKE_LY_PROJECTS":"AutomatedTesting",
-        "CMAKE_TARGET":"AssetProcessorBatch",
-        "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo",
-        "ASSET_PROCESSOR_BINARY": "bin\\profile\\AssetProcessorBatch.exe",
-        "ASSET_PROCESSOR_OPTIONS": "/zeroAnalysisMode --regset=\"/Amazon/AssetProcessor/Settings/Exclude Android/pattern=.*/DiffuseGlobalIllumination/.*precompiledshader\"",
-        "ASSET_PROCESSOR_PLATFORMS":"android"
+      "CONFIGURATION": "profile",
+      "OUTPUT_DIRECTORY": "build\\windows",
+      "CMAKE_OPTIONS": "-DCMAKE_SYSTEM_VERSION=10.0",
+      "CMAKE_LY_PROJECTS": "AutomatedTesting",
+      "CMAKE_TARGET": "AssetProcessorBatch",
+      "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo",
+      "ASSET_PROCESSOR_BINARY": "bin\\profile\\AssetProcessorBatch.exe",
+      "ASSET_PROCESSOR_OPTIONS": "/zeroAnalysisMode --ignoreFutureAssetDatabaseVersionError --regset=\"/Amazon/AssetProcessor/Settings/Exclude Android/pattern=.*/DiffuseGlobalIllumination/.*precompiledshader\"",
```

Comment:
nit: We should pass command line options consistently, prior one is passed by `/` notation

Reply:
I've made that fix in stabilization https://github.com/o3de/o3de/pull/12346 and I'm going to discard this PR because I believe stabilization fixes get merged to development